2200058/Flatten.py

```diff
@@ -0,0 +1,7 @@
+for i in range(10):
+    n=int(input())
+    graph=list(map(int,input().split()))
+    for _ in range(n):
+        graph[graph.index(max(graph))]-=1
+        graph[graph.index(min(graph))]+=1
+    print(f"#{i+1} {graph[graph.index(max(graph))]-graph[graph.index(min(graph))]}")
```

Comment:
문제 상황을 그대로 잘 구현해주셨네요! 👍
```suggestion
    print(f"#{i+1} {max(graph) - min(graph)}")
```
위와 같이 표현하면 더욱 깔끔할 것 같습니다 😄